static/js/components/UserChip_test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+
+import { USER_PROFILE_RESPONSE } from '../constants';
+import R from 'ramda';
+import UserChip from './UserChip';
+import { getPreferredName } from '../util/util';
+import ProfileImage from './ProfileImage';
+
+describe('UserChip', () => {
+  let clone;
+
+  beforeEach(() => {
+    clone = R.clone(USER_PROFILE_RESPONSE);
+  });
+
+  it('should show the preferred name', () => {
+    const chip = shallow(<UserChip profile={clone} />);
```

Comment:
a helper method would be nice here since we're shallow rendering a chip in every single test case. something like:

```javascript
let renderChip = (profile) -> (
    shallow(<UserChip profile={profile} />)
)
```